run.sh

```diff
@@ -0,0 +1,8 @@
+namespace=CtfPerf
+
+for filename in bin/*
+do
+    metricName=$(basename $filename | cut -f 1 -d '.' | sed -r 's/(^|_|-)([a-z])/\U\2/g')
+    echo $(node $filename)
```

Comment:
why do we run the test twice? once here and another time in the next line?
We could run it once, store the result and pass it to the `aws` command. Otherwise the number we show in line 6 isn't necessarily the same as what we send to CloudWatch.

Reply:
accidentally committed